spotbugs/src/main/java/edu/umd/cs/findbugs/io/IO.java

```diff
@@ -167,7 +168,7 @@ public static long copy(Reader in, Writer out, long maxChars)
     }
 
     /**
-     * Close given InputStream, ignoring any resulting exception.
+     * Close given AutoCloseable instance, ignoring any resulting exception.
```

Comment:
It doesn't actually have to be `AutoCloseable`, right? Just `Closeable`.

Reply:
`Closeable` is handled in 190 line.

Problem is that, I replaced `close(Closeable)` with `close(AutoCloseable)` because I thought that it keeps backward compatibility. Yes it keeps compatibility for compiler, but it breaks compatibility for compiled class files; Method invoker expect existence of `close(Closeable)` but there is no method so it throws Exception. See #661 and linked issues for detail.